YuriiYevpak/HW11/HW11.py

```diff
@@ -0,0 +1,40 @@
+# Task 1
+def check_age():
+    try:
+        age = int(input("Enter your age: "))
+        if age < 0:
+            raise ValueError("Age cannot be negative")
+        
+        if age % 2 == 0:
+            print("Your age is even.")
+        else:
+            print("Your age is odd.")
+    
+    except ValueError as ve:
+        print("Error:", ve)
+
+if __name__ == "__main__":
+    check_age()
+
+
+
```

Comment:
good job
look at this version of code:
d = {
    1: 'Monday',
    2: 'Tuesday',
    3: 'Wednesday',
    4: 'Thursday',
    5: 'Friday',
    6: 'Saturday',
    7: 'Sunday'
}

while True:
    try:
        i = int(input('Enter the day of the week: '))
    except ValueError:
        print('You did not enter a number!')
    else:
        print(d.get(i, 'There is no such day of the week!'))